mitodl/mitxpro
# #1345 Receipt Page

WIP: Needs tests and order filter fix
#### Pre-Flight checklist


#### What are the relevant tickets?
#1345 

#### What's this PR do?
Adds a receipt page to the React app for orders that are fulfilled.

#### How should this be manually tested?
Enroll in a course/program and make sure your order is created. Try with/without a coupon, with/without a CyberSource Receipt against the order (can copy the json into local db from RC, model: `ecommerce.models.Receipt`) and the data should show up appropriately.

To view an order's receipt (make sure it's a fulfilled order), go to: `/receipt/<order_id>`

#### Screenshots (if appropriate)
![screencapture-xpro-odl-local-8053-receipt-1-2019-12-04-18_38_00](https://user-images.githubusercontent.com/45350418/70147357-dbc96a00-16c5-11ea-8e0f-6f484f01ca0f.png)
![screencapture-xpro-odl-local-8053-receipt-1-2019-12-03-18_12_22](https://user-images.githubusercontent.com/45350418/70147385-e84dc280-16c5-11ea-81fa-9cb8e7950679.png)
[MIT xPRO _ Receipt-Legal.pdf](https://github.com/mitodl/mitxpro/files/3922077/MIT.xPRO._.Receipt-Legal.pdf)
